iam.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_iam_role" "default" {
```

Comment:
Always use the policy document instead of heredoc. https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html